src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -49,22 +48,18 @@ object SnowflakeTelemetry {
       service
   }
 
-  // The client info telemetry message is only sent one time.
-  def sendClientInfoTelemetryIfNotYet(extraValues: Map[String, String],
-                                      conn: Connection): Unit = {
-    if (!hasClientInfoSent) {
-      SparkConnectorContext.recordConfig()
-      val metric = Utils.getClientInfoJson()
-      for ((key, value) <- extraValues) {
-        metric.put(key, value)
-      }
-      addLog(
-        (TelemetryTypes.SPARK_CLIENT_INFO, metric),
-        System.currentTimeMillis()
-      )
-      send(conn.getTelemetry)
-      hasClientInfoSent = true
+  def sendClientInfoTelemetry(extraValues: Map[String, String],
```

Comment:
To reviewers:
Change to send client info telemetry message per connection instead of per JVM.